components/Trip/index.js

```diff
@@ -2,8 +2,12 @@ import Image from "next/image";
 import Link from "next/link";
 import Location from "../Location";
 import TimePeriod from "../TimePeriod";
+import Counter from "../Counter";
 
 export default function Trip({ title, img, location, timePeriod, _id }) {
+  const startDate = timePeriod.map((timePeriod) => `${timePeriod.startDate}`);
+  const endDate = timePeriod.map((timePeriod) => `${timePeriod.endDate}`);
```

Comment:
Again, why map?

Reply:
We need to enter the data of the array inside of the array and we will keep our schema for now 😅